README.md

```diff
@@ -15,12 +15,13 @@ To start taking screenshots of GLB files
 ```sh
 $ screenshot-glb
 Options:
-  --help        Show help                                              [boolean]
-  --version     Show version number                                    [boolean]
-  -i, --input   Input glTF 2.0 binary (GLB) filepath                  [required]
-  -o, --output  Output PNG screenshot filepath                        [required]
-  -w, --width   Output image width
-  -h, --height  Output image height
+  --help         Show help                                              [boolean]
+  --version      Show version number                                    [boolean]
+  -i, --input    Input glTF 2.0 binary (GLB) filepath                  [required]
+  -o, --output   Output PNG screenshot filepath                        [required]
+  -w, --width    Output image width
+  -h, --height   Output image height
+  -t, --timeout  Timeout length
```

Comment:
In https://github.com/Shopify/screenshot-glb/pull/7 a timeout parameter was added with no way to trigger it from the cli. This adds that.